src/Hng.Domain/Entities/User.cs

```diff
@@ -27,4 +27,5 @@ public class User : EntityBase
     public ICollection<Transaction> Transactions { get; set; } = [];
     public ICollection<Subscription> Subscriptions { get; set; } = [];
     public ICollection<Blog> Blogs { get; set; } = [];
+    public ICollection<UserRole> UsersRoles { get; set; } = [];
```

Comment:
There is no point having this here. Once you add a collection of Roles in User and collection of Users in Role, EntityFramework creates the link table.

Reply:
organisations need to know their users roles